src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,87 @@
+# Integrate your chain
+
+## Prerequisites
+
+- Make sure you have `wasm` enabled.
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 2` to provide your transactions with sufficient gas.
+
+## Devnet information
+
+- Devnet RPC: `http://devnet.rpc.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet.rpc.axelar.dev:1317`
+- Devnet gRPC: `http://devnet.rpc.axelar.dev:9090`
+- Chain ID: `devnet-wasm`
+- Other info can be found in [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json).
+
+## Set up your CosmWasm devnet environment
+
+1. Download the pre-built [`axelard v0.35.5` binary](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) from GitHub.
+1. Generate an account by running `./bin/axelard keys add test`.
```

Comment:
list numbering is wrong

Reply:
These should display correctly on the site itself (see [preview](https://axelar-docs-git-marty-amplifier-integrator-docs-axelar-network.vercel.app/validator/amplifier/integrator-onboarding)) -- using the same number on Markdown files will automatically create an ordered list.

Reply:
I see, interesting, looks weird for someone editing/reviewing though, so might be better to let the editor handle it in the file. But not a strong opinion

Reply:
The main issue I have with hand-numbering is that it's easy to lose count, especially when updating a doc. Much easier to just add another `1`. Imo best way to handle it is to remember that the editor always corrects.